src/runtime_src/core/include/xgq_cmd_vmr.h

```diff
@@ -193,6 +194,8 @@ struct xgq_cmd_vmr_control_payload {
 	uint32_t req_type:8;
 	uint32_t debug_level:3;
 	uint32_t rsvd:21;
+	uint32_t debug_rmgmt_delay;
+	uint32_t debug_vmc_delay;
```

Comment:
There could be more task in future? How about use one field and encode the different VMR apps

Reply:
agree, I can make the change like:
u32 debug_rmgmt:1
u32 debug_vmc:1
u32 debug_rsve:30
u32 debug_value;